packages/patternfly-4/react-core/src/components/AboutModal/index.js

```diff
@@ -1 +0,0 @@
-export { default as AboutModal } from './AboutModal';
```

Comment:
Please rename these files with git instead of deleting and re-adding so we can better preserve the history.

AboutModal.js => AboutModal.tsx
AboutModal.test.js => AboutModal.test.tsx
AboutModalBox.js => AboutModalBox.tsx
AboutModalBoxBrand.js => AboutModalBoxBrand.tsx
AboutModalBoxCloseButton.js => AboutModalBoxCloseButton.tsx
AboutModalBoxContent.js => AboutModalBoxContent.tsx
AboutModalBoxHeader.js => AboutModalBoxHeader.tsx
AboutModalBoxHero.js => AboutModalBoxHero.tsx
AboutModalContainer.js => AboutModalContainer.tsx
index.js => index.ts